system/Debug/Toolbar/Collectors/Views.php

```diff
@@ -73,12 +73,9 @@ class Views extends BaseCollector
      */
     protected $views = [];
 
-    /**
-     * Constructor.
-     */
-    public function __construct()
+    private function getViewer(): void
```

Comment:
probably initViewer() is better naming since it doesn't return anything

Reply:
Exactly!
Fixed.